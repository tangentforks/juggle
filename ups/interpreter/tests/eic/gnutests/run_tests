cg gnu1.c
cg gnu11.c
cg gnu12.c
cg gnu14.c
cg gnu15.c
cg gnu16.c
cg gnu17.c
cg gnu18.c
cg gnu21.c
cg gnu23.c
cg gnu24.c
cg gnu26.c
cg gnu28.c
cg gnu29.c
cg gnu3.c
cg gnu30.c
cx -c gnu4.c
cg gnu4.c
cg gnu6.c
cg gnu7.c
cg gnu8.c
cg -I../../../include t1.c
cg -I../../../include t2.c
cg -I../../../include t3.c
cg -I../../../include t4.c
cg -I../../../include t5.c
cg testcbrt.c
cg testconvert.c
cg testdiv1.c
cg testgofast.c
cg tststruct-big.c
cg tststruct-layout.c
cg tststruct-ret-1.c
cg tststructret.c
