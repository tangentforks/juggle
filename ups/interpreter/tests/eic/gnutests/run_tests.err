gnu11.c, 7: Sizeof may not be applied to a function
	Expression: main
	      type: function(void) returning int
gnu11.c, 8: Warning: Implicit declaration of function printf
cg: Parse of gnu11.c failed
gnu12.c, 28: Warning: Implicit declaration of function printf
gnu12.c, 30: Warning: No prototype in scope for function printf
cg: Error executing gnu12.c: stack overflow
Fatal internal error: bad unary ot (aborting) ...
gnu18.c, 37: Warning: No prototype in scope for function iscmd
gnu21.c, 24: syntax error
cg: Parse of gnu21.c failed
gnu23.c, 53: Conversion to `unsigned short' ignored
	Expression: (unsigned short)(d7)
	      type: unsigned short
gnu24.c, 18: syntax error
cg: Parse of gnu24.c failed
gnu26.c, 51: Warning: Implicit declaration of function printf
gnu26.c, 53: Warning: No prototype in scope for function printf
cg: Error executing gnu26.c: stack overflow
gnu3.c, 29: Warning: Implicit declaration of function printf
gnu30.c, 70: Warning: No prototype in scope for function ExaminePosition
gnu30.c, 72: Warning: Implicit declaration of function printf
gnu30.c, 76: Warning: No prototype in scope for function ExaminePosition
gnu30.c, 78: Warning: Implicit declaration of function printf
gnu30.c, 82: Warning: Implicit declaration of function printf
gnu4.c, 13: Warning: Implicit declaration of function printf
gnu4.c, 14: Warning: No prototype in scope for function printf
gnu4.c, 15: Warning: No prototype in scope for function printf
gnu4.c, 23: Warning: Implicit declaration of function printf
gnu4.c, 13: Warning: Implicit declaration of function printf
gnu4.c, 14: Warning: No prototype in scope for function printf
gnu4.c, 15: Warning: No prototype in scope for function printf
gnu4.c, 23: Warning: Implicit declaration of function printf
gnu6.c, 6: Warning: Implicit declaration of function printf
gnu6.c, 8: Warning: No prototype in scope for function printf
gnu7.c, 5: Warning: Implicit declaration of function printf
gnu7.c, 7: Warning: No prototype in scope for function printf
gnu8.c, 5: Warning: Implicit declaration of function printf
gnu8.c, 7: Warning: No prototype in scope for function printf
Passed test
testconvert.c, 186: Warning: No prototype in scope for function test_integer_to_float
testconvert.c, 187: Warning: No prototype in scope for function test_float_to_integer
tststruct-big.c, 48: Warning: No prototype in scope for function f
tststructret.c, 50: Warning: No prototype in scope for function init
tststructret.c, 54: Warning: No prototype in scope for function init
tststructret.c, 58: Warning: No prototype in scope for function init
