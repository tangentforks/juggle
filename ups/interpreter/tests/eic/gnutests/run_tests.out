Test passed
Test passed
(f)u = 2.14748e+09
(f)(u)0x80000000 = 2.14748e+09
Test passed
Test passed
Test passed
in main()
Test passed.
Test passed.
Test passed
Test passed
Test passed
Test passed
Test passed
Passed
Failure on line 66
Failure on line 68
Failure on line 70
Failure on line 72
Passed
11, 22, 3, 4 : 2.718180
-1, -2, -3, -4 : 2.718180
1, 2, 3, 4 : 3.141500
